### PR TITLE
Logistic Update1

### PR DESCRIPTION
**Pull request by StefArma**
- Updated JNL
- Added to JNL vanilla WotP nato and csat vehicles 
- Activated JNL on missioncrate and enemy ammoboxes

- Changed LOG_suministros: a crate will appear, it can be loaded on trucks and boats, it will tell the delivery location, mission starts only when crate is unloaded, it will stop timer if you load it again, the crate is indestructible if a vehicle explode the crate can be retrived and loaded on another.

Mission tested in SP

### CODE REVIEW

File: Missions/LOG_Suministros.sqf [hunk 1]
```diff
@@ -213,11 +218,14 @@ else
 			};
 	};
 
-_camion setFuel 0;
```
Comment (StefArma) [inline Missions/LOG_Suministros.sqf]: Here i make the crate become empty after the supplies are successfully delivered

File: Missions/LOG_Suministros.sqf [hunk 2]
```diff
@@ -135,26 +135,31 @@ else
...
-		_texto = format ["Bring this truck to %1 and deliver it's supplies to the population",(_this select 0) getVariable "destino"];
-		_texto remoteExecCall ["hint",_this select 2];
-		};
-	}];
 
 [_camion,"Mission Vehicle"] spawn inmuneConvoy;
-if (_dificil) then {reportedVehs pushBack _camion; publicVariable "reportedVehs"};
 
-waitUntil {sleep 1; (not alive _camion) or (dateToNumber date > _fechalimnum) or (_camion distance _posicion < 40)};
+waitUntil {sleep 1; (dateToNumber date > _fechalimnum) or (_camion distance _posicion < 40) and (isNull attachedTo _camion)};
```
Comment (StefArma) [inline Missions/LOG_Suministros.sqf]: here i check that box has to be on ground in order to make mission run

File: Missions/LOG_Suministros.sqf [hunk 3]
```diff
@@ -135,26 +135,31 @@ else
 	_pos = [_posroad, 3, _dirveh + 90] call BIS_Fnc_relPos;
 	};
 
-_camion = "C_Van_01_box_F" createVehicle _pos;
+//Creating the box
+_camion = "Land_PaperBox_01_open_boxes_F" createVehicle _pos;
 _camion allowDamage false;
+_camion call jn_fnc_logistics_addAction;
```
Comment (StefArma) [inline Missions/LOG_Suministros.sqf]: Looking at the box let you know where to deliver it

**Comment by Barbolani77** [comment]
Wow! The thing is telling me how to do this and BTW make intensive use of Jeroen logistics to all the Move Ammobox Content blabla

**Comment by StefArma** [comment]
what is "the thing"? nono it's a working code already, by pushing "merge pull request" it will applied.

Not that intensive, only for ammoboxes and mission box

**Comment by StefArma** [comment]
Already integrated here: https://github.com/A3Antistasi/Antistasi-WotP/commit/73131e3ca2835788b38203719d25b4b98174279c

**Pull request closed without merging**